test/command/feed.spec.ts

```diff
@@ -73,4 +73,37 @@ describe('Test Feed command', () => {
     const length = consoleMessages.length
     expect(consoleMessages[length - 1]).toMatch(/[a-z0-9]{64}/)
   })
+
+  it('should print feed using address only', async () => {
+    // create identity
+    await cli({
+      rootCommandClasses,
+      optionParameters,
+      testArguments: ['identity', 'create', '--identity-name', 'test2', '--password', 'test'],
+    })
+    const config = JSON.parse(readFileSync('test/testconfig/feed.config.json').toString())
+    const address = config?.identities?.test2?.wallet?.address
```

Comment:
please extract the created address from console.log or from the command's instance (if it stores)